satpy/tests/reader_tests/test_fci_l2_nc.py

```diff
@@ -507,3 +507,90 @@ def test_byte_extraction(self):
                                                 })
 
         assert dataset.values == 0
+
+class TestFciL2NCAMVFileHandler(unittest.TestCase):
+    """Test the FciL2NCFileHandler reader."""
+
+    def setUp(self):
```

Comment:
We are going away from python's unittest when possible and use pytest instead. 
```suggestion
class TestFciL2NCAMVFileHandler:
    """Test the FciL2NCFileHandler reader."""

    def setup_method(self):
```